Formula/tailscale.rb

```diff
@@ -0,0 +1,34 @@
+class Tailscale < Formula
+  desc "Easiest, most secure way to use WireGuard and 2FA"
+  homepage "https://tailscale.com"
+  url "https://github.com/tailscale/tailscale.git",
+      tag:      "v1.16.2",
+      revision: "b56ba20549aedc0e56d09dca2552ca62421b2cbd"
+  license "BSD-3-Clause"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X tailscale.com/version.Long=#{version}-#{Utils.git_head}-Homebrew
```

Comment:
can you use our build_dist.sh instead so we don't duplicate this logic?

https://github.com/tailscale/tailscale/blob/main/build_dist.sh

Or at least its `build_dist.sh shellvers` output?

Reply:
I'll move to the shellvers